src/PlaywrightSharp.Tests/Evaluation/FrameEvaluateTests.cs

```diff
@@ -126,5 +126,20 @@ public async Task ShouldNotAllowCrossFrameElementHandlesWhenFramesDoNotScriptEac
             var exception = await Assert.ThrowsAsync<PlaywrightSharpException>(() => Page.EvaluateAsync("body => body.innerHTML", bodyHandle));
             Assert.Contains("Unable to adopt element handle from a different document", exception.Message);
         }
+
+        ///<playwright-file>evaluation.spec.js</playwright-file>
+        ///<playwright-describe>Frame.evaluate</playwright-describe>
+        ///<playwright-it>should return non-empty Node.constructor.name in utility context</playwright-it>
+        [SkipBrowserAndPlatformFact(skipFirefox: true)]
+        public async Task ShouldReturnNonEmptyNodeConstructorNameInUtilityContext()
+        {
+            await Page.GoToAsync(TestConstants.EmptyPage);
+            await FrameUtils.AttachFrameAsync(Page, "frame1", TestConstants.EmptyPage);
+            var frame = Page.Frames[1];
+            var context = await frame.GetUtilityContext();

```

Comment:
Can we add a //TODO here?
//TODO We might want cast frame to a `FrameBase` when implementing this instead of exposing an internal class as an interface.